internal/host/mock_instruction_api.go

```diff
@@ -6,10 +6,9 @@ package host
 
 import (
 	context "context"
-	reflect "reflect"
```

Comment:
This file isn't related to the changes, is it? It looks like you re-generated mocks, but the generator or the IDE changed the order of imports. BTW, some unit tests won't hurt. To test the formatting, and so that any changes don't go unnoticed.

Reply:
If the generator now generates in a different format, then we should start committing it
Also, added some tests, please take a look